packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -40,6 +40,28 @@ export declare class VideoPlayer extends SharedObject<VideoPlayerEvents> {
    */
   currentTime: number;
 
+  /**
+   * The current time of the video as a date. The system calculates this value from the `EXT-X-PROGRAM-DATE-TIME` tag.
+   * > This property is get-only
+   * @platform ios
+   * @platform android
```

Comment:
```suggestion
   * @platform android
   * @platform ios
```

As per our [writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-android-ios-and-web), we prefer referencing platforms in this order: Android, iOS, web. Let's update this to reflect the correct platform order here.